db/migrate/20180504205119_create_books.rb

```diff
@@ -0,0 +1,8 @@
+class CreateBooks < ActiveRecord::Migration[5.1]
+  def change
+    create_table :books do |t|
```

Comment:
I think it's better if you use the same migration for creating the book and setting it's initial values